scripts-available/CDB_Overviews.sql

```diff
@@ -268,44 +268,7 @@ AS $$
     overview_table_name TEXT;
     creation_clause TEXT;
   BEGIN
-    overview_rel := @extschema@._CDB_Overview_Name(reloid, ref_z, overview_z);
-    -- TODO: compute fraction from tolerance_px if not NULL
-    fraction := power(2, 2*(overview_z - ref_z));
-
-    SELECT * FROM @extschema@._cdb_split_table_name(reloid) INTO schema_name, table_name;
-
-    overview_table_name := Format('%I.%I', schema_name, overview_rel);
-    IF has_overview_created THEN
-      RAISE NOTICE 'Sampling reduce stategy deleting and inserting because % has overviews', overview_table_name;
-      EXECUTE Format('DELETE FROM %s;', overview_table_name);
-      creation_clause := Format('INSERT INTO %s', overview_table_name);
-    ELSE
-      RAISE NOTICE 'Sampling reduce stategy creating a new table overview %', overview_table_name;
-      creation_clause := Format('CREATE TABLE %s AS', overview_table_name);
-    END IF;
-
-    -- Estimate number of rows
-    SELECT reltuples, relpages FROM pg_class INTO STRICT class_info
-      WHERE oid = reloid::oid;
-
-    IF class_info.relpages < 2 OR fraction > 0.5 THEN
-      -- We'll avoid possible CDB_RandomTids problems
-      EXECUTE Format('
-        %s SELECT * FROM %s WHERE random() < %s;
-      ', creation_clause, reloid, fraction);
-    ELSE
-      num_samples := ceil(class_info.reltuples*fraction);
-      EXECUTE Format('
-        %1$s SELECT * FROM %2$s
-          WHERE ctid = ANY (
-            ARRAY[
-              (SELECT @extschema@.CDB_RandomTids(''%2$s'', %3$s))
-            ]
-          );
-      ', creation_clause, reloid, num_samples);
-    END IF;
-
-    RETURN Format('%s', overview_table_name)::regclass;
+    RAISE EXCEPTION 'Creating overviews is deprecated';
```

Comment:
Since you've removed the code, I guess it's, technically, "eliminated" rather than "deprecated". 😝